install_config/web_console_customization.adoc

```diff
@@ -506,8 +506,59 @@ endif::[]
 
 endif::[]
 
+[[configuring-the-create-from-url-namespace-whitelist]]
+== Configuring the Create From URL Namespace Whitelist
+
+link:../dev_guide/create_from_url.html[Create from URL] only works with image streams or templates from
```

Comment:
You can remove this link since it points to this same page. If you want to point to an anchor on the page, you can use the format xref:link-name